central/graphql/resolvers/node_components.go

```diff
@@ -6,36 +6,35 @@ import (
 
 	"github.com/graph-gophers/graphql-go"
 	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/graphql/resolvers/loaders"
 	"github.com/stackrox/rox/central/metrics"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/features"
 	pkgMetrics "github.com/stackrox/rox/pkg/metrics"
 	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/scoped"
 	"github.com/stackrox/rox/pkg/utils"
 )
 
 func init() {
 	schema := getBuilder()
 	utils.Must(
 		// NOTE: This list is and should remain alphabetically ordered
-		schema.AddType("NodeComponent", []string{
+		schema.AddExtraResolvers("NodeComponent", []string{
```

Comment:
Add a comment that resolvers for fields of `storage.NodeComponent` are auto-generated and located at `generated.go`. That way readers are not taken by surprise about the missing resolvers.

Reply:
Add a comment to imagecomponent resolver as well.

Reply:
Added comments to both.